src/gui_impl/main_window.cpp

```diff
@@ -44,7 +44,7 @@ MainWindow::MainWindow(QSharedPointer<ApplicationContext> application_context)
     setUnifiedTitleAndToolBarOnMac (true);
     
     QSettings settings("SigViewer");
-    resize(settings.value("MainWindow/size", QSize(900, 800)).toSize());
+    resize(settings.value("MainWindow/size", QSize(1500, 800)).toSize());
```

Comment:
OK, a width of 1200 is a good compromise. Yes, it would be great if the default colors for non-XDF files matched that of XDF files.